build-system/compile/single-pass.js

```diff
@@ -1,790 +0,0 @@
-/**
```

Comment:
Maintaining this file was quite a tedious task. Thanks for cleaning it up!

Reply:
Happy to help.